Pursuit-Core-iOS-Episodes-from-Online/Pursuit-Core-iOS-Episodes-from-Online/ShowViewController.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  ViewController.swift
+//  Pursuit-Core-iOS-Episodes-from-Online
+//
+//  Created by Benjamin Stone on 9/5/19.
+//  Copyright © 2019 Benjamin Stone. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    @IBOutlet weak var showSearchBar: UISearchBar!
+    @IBOutlet weak var showTableView: UITableView!
+    
+    var shows = [Show]() {
```

Comment:
you don't need this shows array if you are not loading something initially, without the user searching for it